tests/drivers/dac/dac_api/src/test_dac.c

```diff
@@ -48,7 +48,8 @@
 	defined(CONFIG_BOARD_SEEEDUINO_XIAO) || \
 	defined(CONFIG_BOARD_ARDUINO_MKRZERO) || \
 	defined(CONFIG_BOARD_ARDUINO_ZERO) || \
-	defined(CONFIG_BOARD_LPCXPRESSO55S36)
+	defined(CONFIG_BOARD_LPCXPRESSO55S36) || \
+	defined(CONFIG_BOARD_FRDM_MCXN947)
```

Comment:
Nits: Can you please add this define next to the other `FRDM` boards so that it's easier to spot?

Reply:
ok